src/Command/CodeStudio/CodeStudioWizardCommand.php

```diff
@@ -161,8 +161,13 @@ private function createProjectAccessToken(array $project, string $projectAccessT
       $this->checklist->completePreviousItem();
     }
     $this->checklist->addItem("Creating access token named <comment>$projectAccessTokenName</comment>");
+    $d = new \DateTime('tomorrow');
+    $formattedDate = $d->format('Y-m-d\TH:i:s\Z');
+    $format = "Y-m-d\TH:i:s\Z";
+    $dateTime = \DateTime::createFromFormat($format, $formattedDate);
```

Comment:
```suggestion
    $dateTime = new \DateTime('tomorrow');
```

Why do you need four lines of code for this? Isn't `$d` identical to `$dateTime`?

Reply:
Actually implemented to get dateTime in specific format but it seems same is the default format. Hence change applied.